sopel/tools/time.py

```diff
@@ -93,7 +86,7 @@ def get_channel_timezone(db, channel):
 
 
 def get_timezone(db=None, config=None, zone=None, nick=None, channel=None):
-    """Find, and return, the approriate timezone
+    """Find, and return, the appropriate timezone
```

Comment:
I'm afraid to git blame this file but I'm pretty sure that's on me... 😁

Reply:
Nope, not this time! It was Embo: a3e6653f2730f7b808f194e02f17c8f691f0955f